packages/extension-webhook/src/index.ts

```diff
@@ -182,16 +182,21 @@ export class Webhook implements Extension {
   }
 
   async onDisconnect(data: onDisconnectPayload) {
-    if (!this.configuration.events.includes(Events.onConnect)) {
+    if (!this.configuration.events.includes(Events.onDisconnect)) {
       return
     }
 
-    await this.sendRequest(Events.onDisconnect, {
-      documentName: data.documentName,
-      requestHeaders: data.requestHeaders,
-      requestParameters: Object.fromEntries(data.requestParameters.entries()),
-      context: data.context,
-    })
+    try {
+        await this.sendRequest(Events.onDisconnect, {
+          documentName: data.documentName,
+          requestHeaders: data.requestHeaders,
+          requestParameters: Object.fromEntries(data.requestParameters.entries()),
+          context: data.context,
+        })
+    } catch (e) {
```

Comment:
I think we should probably move this part to the Hocuspocus instance, so we don't have to take care of potential errors in the hooks

Reply:
Good point, although it would be useful to differentiate between 403 errors for example and 500. I've not had time to delve into the working of the hocuspocus instance yet, but I wonder if it could be done in a way that fixes #508 too?